packages/contracts/hardhat.config.ts

```diff
@@ -108,6 +110,34 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: process.env.ETHERSCAN_API_KEY,
   },
+  dodoc: {
+    runOnCompile: true,
+    exclude: [
+      'Helper_GasMeasurer',
+      'Helper_SimpleProxy',
+      'TestERC20',
+      'TestLib_CrossDomainUtils',
+      'TestLib_OVMCodec',
+      'TestLib_RLPReader',
+      'TestLib_RLPWriter',
+      'TestLib_AddressAliasHelper',
+      'TestLib_MerkleTrie',
+      'TestLib_SecureMerkleTrie',
+      'TestLib_Buffer',
+      'TestLib_Bytes32Utils',
+      'TestLib_BytesUtils',
+      'TestLib_MerkleTree',
+    ],
+  },
+  outputValidator: {
+    runOnCompile: true,
```

Comment:
Remind me how do we run this if this option is switched off in hardhat?

Reply:
Just saw the `docify` command, thanks :)

Reply:
You can now use  - `yarn validateDocs`  👍 
be aware that Dodoc does not support this yet.